### ISSUE TITLE
`@definitelytyped/dtslint` version '0.0.183' requires notNeededPackages.json file to exist

### ISSUE THREAD

**Issue by kamontat** [microsoft/DefinitelyTyped-tools#775]

**Comment by jakebailey** [comment]
Where are you trying to run this tooling outside of DefinitelyTyped itself? It's likely you will be served by a tool like tsd or eslint-plugin-expect-type.

**Comment by kamontat** [comment]
@jakebailey `tsd` you mean [this](https://github.com/tsdjs/tsd) one?

ref: https://github.com/tsdjs/tsd

**Comment by jakebailey** [comment]
Yes, though I guess that tool doesn't check multiple versions. What aspect are you looking for? Just expect type? All of the other stuff is going to be somewhat DT specific, and we have recently greatly changed the layout of DT.

**Comment by kamontat** [comment]
@jakebailey Yes, just expect type.

**Comment by jakebailey** [comment]
Yeah, but unfortunately https://github.com/tsdjs/tsd/issues/47 isn't yet fixed, so it's somewhat of a challenge.

I suspect that if we were to fix the `notNeededPackages` thing, you'd just butt up against a new issue introduced by the huge pnpm monorepo migration, unfortunately.

**Comment by Maxim-Mazurok** [comment]
Hi @jakebailey, I'm having pretty much the same issue, in more details here: https://github.com/microsoft/DefinitelyTyped-tools/issues/850
